infra/charts/feast-core/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+description: "Feast Core: Feature registry for Feast."
+name: feast-core
+version: 0.25.0
+appVersion: v0.9.2
```

Comment:
is there a reason why this is still 0.9.2? same goes for feast-serving/Chart.yaml

Reply:
Nope, just haven't updated it. What should the version number be?

Reply:
let's do appVersion = version = 0.25.0?